packages/components/src/custom-select-control/index.js

```diff
@@ -103,6 +105,20 @@ export default function CustomSelectControl( props ) {
 
 	const [ isFocused, setIsFocused ] = useState( false );
 
+	function handleOnFocus( e ) {
+		setIsFocused( true );
+		if ( typeof onFocus === 'function' ) {
+			onFocus( e );
+		}
```

Comment:
```suggestion
		onFocus?.( e );
```

I'm seeing us use optional chaining more now when handlers are optional props. Should we use that here to make things more concise?

Reply:
ah, nice, I hadn't noticed that use of optional chaining before, done.